google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -789,6 +809,22 @@ Builder setUseMultiplexedSession(boolean useMultiplexedSession) {
       return this;
     }
 
+    /**
+     * Sets whether the client should use multiplexed session for blind write or not. If set to
+     * true, the client optimises and runs multiple applicable requests concurrently on a single
+     * session. A single multiplexed session is sufficient to handle all concurrent traffic.
+     *
+     * <p>When set to false, the client uses the regular session cached in the session pool for
+     * running 1 concurrent transaction per session. We require to provision sufficient sessions by
+     * making use of {@link SessionPoolOptions#minSessions} and {@link
+     * SessionPoolOptions#maxSessions} based on the traffic load. Failing to do so will result in
+     * higher latencies.
+     */
```

Comment:
I think that it is better to remove this whole block of documentation, and instead just mark this method as `@InternalApi` and `@VisibleForTesting`, and maybe also add one bit of Javadoc that says something like this:

```
This method enables multiplexed sessions for blind writes. This method will be removed in the future when multiplexed sessions has been made the default for all operations.
```

(The above to prevent users from asking why it is not public / how they can enable it / etc.)